src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/rest/Comments.java

```diff
@@ -1,54 +0,0 @@
-/*
- *  The MIT License
- *
- *  Copyright 2013 Jyrki Puttonen. All rights reserved.
- *  Copyright 2013 Sony Mobile Communications AB. All rights reserved.
- *
- *  Permission is hereby granted, free of charge, to any person obtaining a copy
- *  of this software and associated documentation files (the "Software"), to deal
- *  in the Software without restriction, including without limitation the rights
- *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- *  copies of the Software, and to permit persons to whom the Software is
- *  furnished to do so, subject to the following conditions:
- *
- *  The above copyright notice and this permission notice shall be included in
- *  all copies or substantial portions of the Software.
- *
- *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- *  THE SOFTWARE.
- */
-package com.sonymobile.tools.gerrit.gerritevents.dto.rest;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-/**
- * A list of {@link LineComment}.
- * Not used.
- */
-public class Comments {
```

Comment:
Hmm, though I like your design of putting the line comments inside the commented file object better than what it was before, I don't like removing classes and breaking binary compatibility.
Is there some way we can keep this class around but deprecate it and let ReviewInput still accept it as a deprecated way? It would make me sleep better :)